ui/pages/settings/networks-tab/networks-form/use-safe-chains.ts

```diff
@@ -49,8 +49,12 @@ export const rpcIdentifierUtility = (
 
   for (const chain of safeChains) {
     for (const rpc of chain.rpc) {
-      if (host === new URL(rpc).host) {
-        return host;
+      try {
+        if (host === new URL(rpc).host) {
+          return host;
+        }
+      } catch {
+        continue;
```

Comment:
curious: why was this `try` needed?

Reply:
One of the rpc URLs was not a valid URL (they're sourced from `https://chainid.network/chains.json`).  It was something like `https://foo.bar, wss://foo.bar`.  This threw an error and prevented the rest of the loop from running.